web/impact/impact/graphql/types/expert_profile_type.py

```diff
@@ -88,3 +98,10 @@ def _get_user_programs(user):
     )
     return Program.objects.filter(
         programrole__in=user_program_roles_as_participant).distinct()
+
+
+def _get_mentees(expert_id):
+    user_id = ExpertProfile.objects.get(id=expert_id).user_id
+    relationships = StartupMentorRelationship.objects.filter(
+        mentor_id=user_id).all()
+    return [r.startup_mentor_tracking.startup for r in relationships]
```

Comment:
A more django-esque way to write this function would be:

```
def resolve_previous_mentees(self, info, **kwargs):
    return _get_mentees(self.user, ENDED_PROGRAM_STATUS)

def resolve_current_mentees(self, info, **kwargs):
    return _get_mentees(self.user, ACTIVE_PROGRAM_STATUS)

def _get_mentees(user, program_status): # pass in self.user
    return Startup.objects.filter(startupmentortrackingrecord__startupmentorrelationship__mentor=self.user, startupmentortrackingrecord__program__program_status=program_status)
```

Advantages of this approach:
- You return a queryset instead of a list, which keeps you in the django world (which is good because e.g. lazy evaluation, extra filtering later on if needed, and just generally more consistent interfaces).
- you filter for the program_status you want in the database layer instead of python, which is more efficient.
- you filter for the program_status directly on the mentor_tracking object, which gives you the status of the relevant program, instead of the first program for that startup, and this avoids errors.
- you return startups objects right away, so no need to iterate over the results once again.